scripts/cluster/create_one_node_cluster.sh

```diff
@@ -26,9 +26,15 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 ROOT="$( cd $DIR && cd .. && cd .. && pwd)"
 
 # Create kubelet service
-$DIR/setup_worker_kubelet.sh
+$DIR/setup_worker_kubelet.sh $1
```

Comment:
please create a named variable instead of $1. it's more readable

Reply:
done